File: demo/java/custom-java/src/main/java/com/baidu/bce/models/Point.java
package com.baidu.bce.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Point {
    /**
     * 点位标识符
     */
    public static final int INDEX_TEMPERATURE = 0;
    public static final int INDEX_HUMIDITY = 1;
    public static final int INDEX_PRESSURE = 2;

    private float temperature;
    private float humidity;
    private float pressure;
}

File: demo/csharp/custom-csharp/custom-csharp/Models/Point.cs
namespace Buildin.Csharp.Models
{
    public class Point
    {
        public const int INDEX_TEMPERATURE = 0;
        public const int INDEX_HUMIDITY = 1;
        public const int INDEX_PRESSURE = 2;

        public double Temperature { get; set; } = 0.0;
        public double Humidity { get; set; } = 0.0;
        public double Pressure { get; set; } = 0.0;

        public Point() { }

        public Point(double temperature, double humidity, double pressure)
        {
            Temperature = temperature;
            Humidity = humidity;
            Pressure = pressure;
        }
    }
}

